generated/nifgen/nifgen/nidevice_pb2.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
I see `_grpc.py` is prefixed with underscore denoting it's an internal implementation detail.
This file and others should follow suit.

Reply:
This is how they're generated by protoc. Perhaps a better option would be to put them in an underscore-prefixed sub-package?